### ISSUE TITLE
Refactor shaka-player to use CML's ID3 utilities

### ISSUE THREAD

**Issue by littlespex** [streaming-video-technology-alliance/common-media-library#59]
**Is your feature request related to a problem? Please describe.**
The ID3 utility functions in CML are based in part on the ones in `shaka-player`. Now that the ID3 functionality has been consolidated from the player libraries, the players should be refactored to use CML's version.

**Describe the solution you'd like**
Refactor `shaka-player` to use CML's ID3 utilities.

**Comment by fyoungqualabs** [comment]
Shaka uses google closure library to manage it’s dependencies, now turns out Google Closure Library isn’t compatible with ES6, so we can’t import CML through this method. 
On top of that, there’s another situation worth mentioning: Google Clousure Libray is being deprecated on August 1, 2024.
As a consequence of this we think our best option is to keep this on hold for now(instead of trying to come up with strategies that will not be very useful after Google Clousure Library deprecation), until Shaka figures out what strategy is going to take in this matter.
@littlespex what do you think about this?

**Comment by littlespex** [comment]
I agree this should be kept on hold until Shaka migrates away from Closure.
